.env.example

```diff
@@ -5,6 +5,8 @@ HTTP_PORT=64100
 HTTPS_PORT=64101
 MINIO_PORT=64140
 PHPMYADMIN_PORT=64180
+MAILCATCHER_PORT_SMTP=64125
+MAILCATCHER_PORT_HTTP=64126
```

Comment:
I'd like to use Mailhog instead of Mailcatcher if there are no objections.
 I'm open for suggestions though, are there specific benefits compared to Mailhog?